go.mod

```diff
@@ -0,0 +1,46 @@
+module github.com/giantswarm/release-operator
+
+go 1.13
+
+require (
+	github.com/giantswarm/apiextensions v0.2.0
+	github.com/giantswarm/k8sclient v0.2.0
+	github.com/giantswarm/microendpoint v0.2.0
+	github.com/giantswarm/microerror v0.2.0
+	github.com/giantswarm/microkit v0.2.0
+	github.com/giantswarm/micrologger v0.3.1
+	github.com/giantswarm/operatorkit v0.2.0
+	github.com/google/go-cmp v0.4.0
+	github.com/spf13/jwalterweatherman v1.1.0 // indirect
+	github.com/spf13/pflag v1.0.5 // indirect
+	github.com/spf13/viper v1.6.2
+	golang.org/x/net v0.0.0-20191004110552-13f9640d40b9 // indirect
```

Comment:
Why is this weird import here?

Reply:
Don't know. I tried to drop it but `go mod tidy` adds it.